src/platforms/px4_tasks.h

```diff
@@ -99,6 +101,53 @@ typedef struct {
 #error "No target OS defined"
 #endif
 
+// Fast drivers - they need to run as quickly as possible to minimize control
+// latency.
+#define SCHED_PRIORITY_FAST_DRIVER		(SCHED_PRIORITY_MAX - 0)
+
+// Attitude controllers typically are in a blocking wait on driver data
+// they should be the first to run on an update, using the current sensor
+// data and the *previous* attitude reference from the position controller
+// which typically runs at a slower rate
+#define SCHED_PRIORITY_ATTITUDE_CONTROL		(SCHED_PRIORITY_MAX - 4)
+
+// Actuator outputs should run before right after the attitude controller
+// updated
+#define SCHED_PRIORITY_ACTUATOR_OUTPUTS		(SCHED_PRIORITY_MAX - 4)
```

Comment:
This might require an ACTUATOR_OUTPUTS_SLOW and _FAST option. First tests show that the system response on e.g. the shell is noticeable more sloppy in the new configuration. Need to figure out which processes are the main contributors to that.